code/__DEFINES/flags.dm

```diff
@@ -30,6 +30,7 @@
 #define THICKMATERIAL 8192		//prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body. (NOTE: flag shared with BLOCK_GAS_SMOKE_EFFECT)
 #define DROPDEL			16384 // When dropped, it calls qdel on itself
 #define HOLOGRAM		32768	// HOlodeck shit should not be used in any fucking things
+#define OVERLAY_QUEUED  65536	//atom queued to SSoverlay

```

Comment:
You can replace FPRINT flag it's unused. In general some of these need moving , these should be most generic flags only.